Sources/ShortcutRecorder/SRRecorderControl.m

```diff
@@ -928,6 +945,16 @@ - (void)playAlert
     NSBeep();
 }
 
+- (void)showSequoiaAlert
+{
+    NSAlert *sequoiaAlert = [[NSAlert alloc] init];
+    [sequoiaAlert setMessageText:SRLoc(@"Shortcut not valid on macOS Sequoia")];
+    [sequoiaAlert setInformativeText:SRLoc(@"Sequoia cannot use a shortcut with just opt or shift as modifiers.  Opt and shift are valid, but must be combined with cmd or ctrl.")];
```

Comment:
Wording suggestions:

- Shortcut not valid
- Shortcuts must include the command or control key. Shortcuts with only option or shift are not allowed by macOS.

Reply:
👍🏻